packages/client/src/v2-events/features/events/registered-fields/Address.tsx

```diff
@@ -241,11 +242,14 @@ function AddressInput(props: Props) {
     (location) => location.locationType === 'ADMIN_STRUCTURE'
   )
 
-  const administrativeArea = value?.administrativeArea
+  const administrativeArea =
+    value?.addressType === AddressType.DOMESTIC
+      ? value.administrativeArea
+      : undefined
```

Comment:
Perhaps we can use a helper to extract `administrativeArea` from `value`? As this is being done quite a few times